nav2_system_tests/src/error_codes/test_error_codes.py

```diff
@@ -26,6 +26,7 @@
 
 def main(argv=sys.argv[1:]):
     rclpy.init()
+    time.sleep(10)
```

Comment:
? 10 seconds is alot to add to a test. What is this waiting for? Can this be reduced or based on an event?

Reply:
It is just waiting for the whole stack to come up. The same thing is done for the nav though poses test. 

https://github.com/ros-planning/navigation2/blob/main/nav2_system_tests/src/system/nav_through_poses_tester_node.py#L321

This should be resolved with the life cycle changes for python.